src/win/tty.c

```diff
@@ -153,6 +165,9 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, uv_file fd, int readable) {
     fd = -1;
   }
 
+  if (vterm == UNCHECKED && !readable)
```

Comment:
given that this is a global variable and multiple threads can potentially be creating tty handles, there is race condition here.